handout/summary.md

```diff
@@ -0,0 +1,27 @@
+# Summary
```

Comment:
* Rename Summary to Glossary,
* Also rename file to `glossary.md`
* Update the main `summary.md` that acts as the TOC, and add a link to this glossary just above 'Other Resources'